pkgs/by-name/uv/uv/package.nix

```diff
@@ -66,6 +66,15 @@ rustPlatform.buildRustPackage (finalAttrs: {
 
   meta = {
     description = "Extremely fast Python package installer and resolver, written in Rust";
+    longDescription = ''
+      `uv` manages project dependencies and environments, with support for lockfiles, workspaces, and more.
+
+      When running `uv` on NixOS systems, make sure to use `UV_PYTHON` or `--python`, where applicable, to pass a path to a compatible interpreter
+      via dev-shell (e.g. `UV_PYTHON=''${pkgs.python313}`), .env file, or from the command line, as NixOS cannot run dynamically
+      linked executables intended for generic Linux environments out of the box.
```

Comment:
This could be a bit misleading.  Just setting the interpreter is not enough as uv can download other binaries that are dynamically linked (for manylinux*).

Also this seems more appropriate for a section in Nixpkgs manual.

Reply:
It was my understanding that `UV_PYTHON` leads to `uv` not downloading other binaries.
There was a lengthy discussion here: https://github.com/NixOS/nixpkgs/pull/429813